CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Eclipse Lyo changelog
 
-## [Unreleased]
+## [5.1.1]
```

Comment:
I would rename 5.1.0 to 5.1.1 and instead add an entry below 5.1.1 like this

```
**[5.1.0] - YANKED**

The release was yanked due to a problem with the publication of release artifacts to Maven Central. Please use `5.1.1.Final` instead.
```

Reply:
@berezovskyi ! Good now?